### PR TITLE
Main content changes

### PR DESCRIPTION
**Pull request by itsmepiyush2**
Changed certain texts in the main content as suggested by @palash25 bhaiya and @shivank01 bhaiya. Please let me know if I have carried out things correctly. I'm still learning.

### CODE REVIEW

**Comment by palash25** [comment]
> @palash25 bhaiya and @shivank01 bhaiya. 

no need of bhaiya's just using our github handles doesn't mean that you are being disrespectful. We are all peers I don't want anyone to use bhaiya or didi on github and I am even cool with us being on a first name basis in person too so chill :wink: 

> Please let me know if I have carried out things correctly

You haven't I am guessing that you checked out this branch from the branch of your 1st PR didn't you?

**Comment by palash25** [comment]
I guess that is the reason why this commit is showing up 9002e66710375df78e641b87057f9814d60152f4

You will need to delete this commit and just push the tip of your branch which is currently your second commit.

To avoid this in the future always make a new branch from your master and not any other brach (this creates a copy of the master branch so only one commit would show up in that case)

Also read this please https://github.com/p-society/meta @itsmepiyush2

File: _trophy/2.2017.md [hunk 1]
```diff
@@ -4,27 +4,27 @@ achievements_students:
...
+    - Prateek Pattnaik, Nawed Imroze and Bibhu Pala are selected for ACM ICPC Amritapuri Regionals
   - |
-    - Team PiedPiper (someone please fill in the names of the members) won the first prize in Smart India Hackathon.
+    - Team PiedPiper (someone please fill in the names of the members) win the first prize in Smart India Hackathon.
   - |
-    - Team "The Web Samaritans" won the second runners-up in Smart India Hackathon(members:Abhijeet Panda, Anuva Acharya, Sheikh  - Sohail, Sami Ur Rehman, Rohan Panda, Shubhankar Mohapatra)
+    - Team "The Web Samaritans" (members: Abhijeet Panda, Anuva Acharya, Sheikh - Sohail, Sami Ur Rehman, Rohan Panda, Shubhankar Mohapatra) win the second runners-up in Smart India Hackathon
   - |
-    - Another team were the finalist in GE Healthhack can someone fill in their names please (cc @sr6033 @shibasisp )
+    - Another team become finalist in GE Healthhack can someone fill in their names please (cc @sr6033 @shibasisp )
```
Comment (palash25) [inline _trophy/2.2017.md]: We need this removed `(cc @sr6033 @shibasisp )` I guess some careless contributor just copy pasted the things I had written on an issue thread just remove any *cc's* or @ mentions you find @itsmepiyush2

File: _config.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 title: P-society
 email: psocietyiiitbh@gmail.com
-description: We are a bunch of undergrads who like to dabble into all things in computer science from
-             web dev to machine learning and from distributed systems to blockchain,
-             helping others along the way in kick-starting their journey of becoming a developer.
+description: We are a bunch of undergraduate students who like to dabble into all things computer science from
```
Comment (palash25) [inline _config.yml]: > undergraduate students

that's quite a mouthful. what's wrong with undergrads?

File: _config.yml [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
+             web development, to machine learning, to distributed systems, to blockchain,
```
Comment (palash25) [inline _config.yml]: so many `to's` can you phrase it any other way?

File: _trophy/2.2017.md [hunk 4]
```diff
@@ -4,27 +4,27 @@ achievements_students:
...
   - |
-    - Shivank Pathak becomes the 4th runner up in Kronothon.
+    - Shivank Pathak becomes the 4th runner up in Kronothon
   - |
     - Aman Kumar Gupta secures National Rank 35 in TechGig Code Gladiators
   - |
-    - Shivank Pathak and Abhishek Patel selected as finalist in GDG BBSR Devfest Hackathon
+    - Shivank Pathak and Abhishek Patel are selected as finalists in GDG BBSR Devfest Hackathon
   - |
     - There was another team in the GDG finals was it yours @hdidwania ?
```
Comment (shivank01) [inline _trophy/2.2017.md]: Change it to "Himanshu Didwania and Abhipsha Das are selected as finalists in GDG BBSR Devfest Hackathon"

File: _trophy/2.2017.md (hunk 1 above)
Comment (itsmepiyush2) [reply _trophy/2.2017.md]: Okay

File: _config.yml (hunk 2 above)
Comment (itsmepiyush2) [reply _config.yml]: Okay will leave it at undergrads. Nothing wrong!

File: _config.yml (hunk 3 above)
Comment (itsmepiyush2) [reply _config.yml]: I'll look into it.

File: _trophy/2.2017.md (hunk 4 above)
Comment (itsmepiyush2) [reply _trophy/2.2017.md]: Okay

**Comment by palash25** [comment]
were you able to remove your previous commit @itsmepiyush2 ?

**Comment by itsmepiyush2** [comment]
I tried to revert the commit but I faced the following error.
screen shot 2018-08-20 at 7 17 12 pm

**Comment by palash25** [comment]
I am busy right now will answer by tomorrow

**Comment by knrt10** [comment]
@itsmepiyush2 closing this PR. Please make new PR with the same changes.
Follow these steps after opening your cloned repo of p-soc:-
1) git checkout master
2) git remote add upstream https://github.com/p-society/p-society.github.io.git
3) git fetch upstream
4) git rebase origin/master
5) git checkout -b newBranch(this can be anything you want).
Copy old changes to files.
6) git add -A
7) git commit -m "your commit message"
8) git push origin newBranch(branch name from the step 5)
Now make a PR comparing your remote to the master of p-society

**Pull request closed without merging**